activesupport/test/core_ext/integer_ext_test.rb

```diff
@@ -31,11 +31,11 @@ def test_ordinal
   def test_positive
     assert 1.positive?
     assert_not 0.positive?
-    assert_not -1.positive?
+    assert_not (-1).positive?
   end
 
   def test_negative
-    assert -1.negative?
+    assert (-1).negative?
```

Comment:
I think it might be better to use like this `assert(-1.negative?)` instead, wdyt?